cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -551,4 +552,24 @@ public static String getJavaSystemPropertiesToString(List<SystemProperty> javaSy
         }
         return String.join(" ", javaSystemPropertiesList);
     }
+
+    public static void removeEmptyValuesFromTolerations(List<Toleration> tolerations) {
+        if (tolerations != null) {
+            tolerations.stream().filter(toleration -> toleration.getValue() != null && toleration.getValue().isEmpty()).forEach(emptyValTol -> emptyValTol.setValue(null));
+        }
+    }
+
+    public static List<Toleration> tolerations(String tolerations, String templateTolerations, PodTemplate podTemplate, List<Toleration> tolerationList) {
```

Comment:
Think about the parameter order. Why is it sensible to put `tolerations` first and `tolerationList` last, when they're to do with the same thing (the former is describing the path at which we found the latter). They belong next to each other, imho.

Reply:
I wanted to couple `String` parameters which are used one right after the another one.

Reply:
When writing a method signature you should usually prioritise how it will look to the caller. The caller doesn't know (or care) in what order the parameters will be used. They just have some arguments, and it makes much more sense to them to pass the path and the value got from that path together.